addons/website/static/src/core/colibri.js

```diff
@@ -82,10 +82,18 @@ export class Colibri {
                 this.updateContent();
             }
         };
+        const removeListeners = [];
         for (const node of nodes) {
             node.addEventListener(event, handler, options);
-            this.cleanups.push(() => node.removeEventListener(event, handler, options));
+            const removeListener = () => node.removeEventListener(event, handler, options);
+            this.cleanups.push(removeListener);
+            const currIdx = this.cleanups.length - 1;
+            removeListeners.push(() => {
+                this.cleanups.splice(currIdx, 1);
```

Comment:
maybe not needed, idk

Reply:
`currIdx` will not be stable if this mecanism is used to remove them in a custom sequence

Reply:
:facepalm: of course